src/scripts/oakItems/OakItems.ts

```diff
@@ -18,14 +23,26 @@ class OakItems implements Feature {
-            new OakItem(OakItems.OakItem.Poison_Barb, 'Poison Barb', [1.25, 1.30, 1.35, 1.40, 1.45, 1.50], 1, true, 40, 'Clicks do more damage', 3),
-            new OakItem(OakItems.OakItem.Exp_Share, 'Exp Share', [1.15, 1.18, 1.21, 1.24, 1.27, 1.30], 1, true, 50, 'Gain more exp from battling', 1),
-            new OakItem(OakItems.OakItem.Sprayduck, 'Sprayduck', [1.25, 1.30, 1.35, 1.40, 1.45, 1.50], 1, false, 60, 'Makes your berries grow faster', 3),
-            new OakItem(OakItems.OakItem.Shiny_Charm, 'Shiny Charm', [1.50, 1.60, 1.70, 1.80, 1.90, 2.00], 1, true, 70, 'Encounter shinies more often', 150),
-            new OakItem(OakItems.OakItem.Blaze_Cassette, 'Blaze Cassette', [1.50, 1.60, 1.70, 1.80, 1.90, 2.00], 1, false, 80, 'Hatch eggs faster', 10),
-            new OakItem(OakItems.OakItem.Cell_Battery, 'Cell Battery', [1.5, 1.6, 1.7, 1.8, 1.9, 2], 1, false, 90, 'More passive mining energy regen', 20),
+            new OakItem(OakItems.OakItem.Magic_Ball, 'Magic Ball', 'Gives a bonus to your catchrate',
+                true, [5, 6, 7, 8, 9, 10], 0, 20, 2),
+            new OakItem(OakItems.OakItem.Amulet_Coin, 'Amulet Coin', 'Gain more coins from battling',
+                true, [1.25, 1.30, 1.35, 1.40, 1.45, 1.50], 1, 30, 1),
+            new OakItem(OakItems.OakItem.Poison_Barb, 'Poison Barb', 'Clicks do more damage',
+                true, [1.25, 1.30, 1.35, 1.40, 1.45, 1.50], 1, 40, 3),
+            new OakItem(OakItems.OakItem.Exp_Share, 'EXP Share', 'Gain more exp from battling',
+                true, [1.15, 1.18, 1.21, 1.24, 1.27, 1.30], 1, 50, 1),
+            new OakItem(OakItems.OakItem.Sprayduck, 'Sprayduck', 'Makes your berries grow faster',
+                false, [1.25, 1.30, 1.35, 1.40, 1.45, 1.50], 1, 60, 3),
+            new OakItem(OakItems.OakItem.Shiny_Charm, 'Shiny Charm', 'Encounter shinies more often',
+                true, [1.50, 1.60, 1.70, 1.80, 1.90, 2.00], 1, 70, 150),
+            new OakItem(OakItems.OakItem.Blaze_Cassette, 'Blaze Cassette', 'Hatch eggs faster',
+                false, [1.50, 1.60, 1.70, 1.80, 1.90, 2.00], 1, 80, 10),
+            new OakItem(OakItems.OakItem.Cell_Battery, 'Cell Battery', 'More passive mining energy regen',
+                false, [1.5, 1.6, 1.7, 1.8, 1.9, 2], 1, 90, 20),
+            new BoughtOakItem(OakItems.OakItem.Squirtbottle, 'Squirtbottle', 'Increases the chance of berry mutations', 'Johto Berry Master',
+                true, [1.15, 1.3, 1.45, 1.6, 1.75], 1, 5, undefined, undefined, AmountFactory.createArray([500, 1000, 2000, 3000, 5000], GameConstants.Currency.farmPoint)),
+            new BoughtOakItem(OakItems.OakItem.Sprinklotad, 'Sprinklotad', 'Increases the chance of berry replants', 'Hoenn Berry Master',
+                true, [1.5, 1.75, 2, 2.25, 2.5], 1, 7, undefined, undefined, AmountFactory.createArray([500, 1000, 2000, 3000, 5000], GameConstants.Currency.farmPoint)),
```

Comment:
Are these 2 the wrong way around?

As with the Sprinklotad, if the multiplier is 2+ then it's a 100% replant rate with all the Kanto berries.
Also seems to be able upgrade way too fast IMO.
```suggestion
            new BoughtOakItem(OakItems.OakItem.Squirtbottle, 'Squirtbottle', 'Increases the chance of berry mutations', 'Johto Berry Master',
                true, [1.15, 1.3, 1.45, 1.6, 1.75], 1, 5, undefined, undefined, AmountFactory.createArray([500, 1000, 2000, 3000, 5000], GameConstants.Currency.farmPoint)),
            new BoughtOakItem(OakItems.OakItem.Sprinklotad, 'Sprinklotad', 'Increases the chance of berry replants', 'Hoenn Berry Master',
                true, [1.5, 1.75, 2, 2.25, 2.5], 1, 7, undefined, undefined, AmountFactory.createArray([500, 1000, 2000, 3000, 5000], GameConstants.Currency.farmPoint)),
```

Reply:
The Oak Items were implemented before I fully set all the properties for the berries. I'm always up for rebalancing (mostly because I didn't do that many test runs to check if they were balanced to being with). 

Experience required probably needs to go way up, especially since there are a lot more berries to work with now, so we can extend the Berry mechanic way into the next regions (aka slower growth).

We do want to balance the Sprinklotad to ensure that at the end game the player can have 100 replant so the farm can become fully passive. Whether that means 2x, or something less with Replant aura berries around is to be decided still.

Reply:
IMO it shouldn't ever be a 100% replant chance from just the Oak Item, especially with how quickly the oak items level up.

Reply:
We could probably also increase the upgrade cost, as that also seems quite low.

Reply:
Maybe these amount would work okay, What do you think?
```suggestion
            new BoughtOakItem(OakItems.OakItem.Squirtbottle, 'Squirtbottle', 'Increases the chance of berry mutations', 'Johto Berry Master',
                true, [1.5, 1.75, 2, 2.25, 2.5], 1, 10, undefined, undefined, AmountFactory.createArray([1000, 2000, 5000, 10000, 20000], GameConstants.Currency.farmPoint)),
            new BoughtOakItem(OakItems.OakItem.Sprinklotad, 'Sprinklotad', 'Increases the chance of berry replants', 'Hoenn Berry Master',
                true, [1.15, 1.3, 1.45, 1.6, 1.75], 1, 2, undefined, undefined, AmountFactory.createArray([1000, 2000, 5000, 10000, 20000], GameConstants.Currency.farmPoint)),
```